### PR TITLE
docs: Update examples for all scenarios

### PR DESCRIPTION
**Pull request by scottford-lw**
This PR updates all example scenarios with updated READMEs and links back to Lacework support docs.

Signed-off-by: Scott Ford <scott.ford@lacework.net>

### CODE REVIEW

File: examples/complete-cloudtrail/README.md [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+  }
+}
+
+provider "aws" {}
+
+provider "lacework" {}
+
+module "aws_cloudtrail" {
+  source  = "lacework/cloudtrail/aws"
+  version = "~> 0.1.3"
```
Comment (afiune) [inline examples/complete-cloudtrail/README.md:28]:
```suggestion
  version = "~> 0.1.5"
```

File: examples/complete-cloudtrail-access-logs/README.md [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+}
+
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+module "aws_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/complete-cloudtrail-access-logs/README.md:25]: The examples inside the README.md should have the actual source and version:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

File: examples/complete-cloudtrail-versioning/README.md [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+ 
+```
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+module "aws_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/complete-cloudtrail-versioning/README.md:18]:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

File: examples/complete-cloudtrail/main.tf [hunk 4]
```diff
@@ -1,15 +1,24 @@
...
 }
 
+provider "aws" {}
+
+provider "lacework" {}
+
 module "aws_cloudtrail" {
-  source = "../../"
+  source  = "lacework/cloudtrail/aws"
+  version = "~> 0.1.3"
```
Comment (afiune) [inline examples/complete-cloudtrail/main.tf:16]: These are living tests, we need them to point to the local directory:
```suggestion
  source = "../../"
```

File: examples/consolidated-cloudtrail-multiple-lacework-tenants/README.md [hunk 5]
```diff
@@ -1,9 +1,82 @@
...
+}
+
+provider "aws" {
+  profile = "main-account"
+  alias   = "main-account"
+  region  = "us-west-2"
+}
+
+module "main_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/consolidated-cloudtrail-multiple-lacework-tenants/README.md:25]:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

File: examples/consolidated-cloudtrail/README.md [hunk 6]
```diff
@@ -1,5 +1,64 @@
...
+}
+
+provider "aws" {
+  profile = "main-account"
+  alias   = "main-account"
+  region  = "us-west-2"
+}
+
+module "main_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/consolidated-cloudtrail/README.md:20]:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

File: examples/existing-cloudtrail-iam-role/README.md [hunk 7]
```diff
@@ -12,3 +13,29 @@ the fields required for this example are:
...
+
+```
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+module "aws_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/existing-cloudtrail-iam-role/README.md:25]:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

File: examples/existing-cloudtrail/README.md [hunk 8]
```diff
@@ -15,3 +14,23 @@ The fields required for this example are:
...
+
+```
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+module "aws_cloudtrail" {
+  source = "../../"
```
Comment (afiune) [inline examples/existing-cloudtrail/README.md:26]:
```suggestion
  source  = "lacework/cloudtrail/aws"
  version = "~> 0.1.5"
```

**Review by afiune — CHANGES_REQUESTED** [review]

**Comment by scottford-lw** [comment]
@afiune this should be good to go!

**Comment by scottford-lw** [comment]
@afiune I spoke too soon. It looks like the `main.tf` cannot have a `required_providers` if we include a `versions.tf`

**Comment by scottford-lw** [comment]
@afiune this is good to go now.

File: examples/complete-cloudtrail/README.md [hunk 9]
```diff
@@ -1,11 +1,38 @@
...
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+      version = "~> 0.2.7"
+    }
+  }
+}
+
+provider "aws" {}
```
Comment (afiune) [inline examples/complete-cloudtrail/README.md:22]:
```suggestion
provider "aws" {
  region = "us-west-2"
}
```

File: examples/complete-cloudtrail/main.tf [hunk 10]
```diff
@@ -1,8 +1,6 @@
-provider "lacework" {}
+provider "aws" {}
```
Comment (afiune) [inline examples/complete-cloudtrail/main.tf:1]:
```suggestion
provider "aws" {
  region = "us-west-2"
}
```

File: examples/complete-cloudtrail/main.tf [hunk 11]
```diff
@@ -12,4 +10,5 @@ module "aws_cloudtrail" {
   bucket_enable_encryption   = true
   bucket_enable_logs         = true
   bucket_enable_versioning   = true
+
```
Comment (afiune) [inline examples/complete-cloudtrail/main.tf:13]:
```suggestion
```

**Review by afiune — APPROVED** [review]
tenor-27190589

**Pull request merged**